e2etest/declarativeTestFiles.go

```diff
@@ -220,13 +220,15 @@ func (t *testObject) isRootFolder() bool {
 // It is to be used ONLY as parameters to the f() and folder() methods.
 // It is not used in other parts of the code, since the other parts use the testObject instances that are created
 // from
+//nolint:unused
 type withPropertyProvider interface {
```

Comment:
If these interfaces are not used we can remove them.

Reply:
I want to schedule a meeting to discuss these unused pieces just so everyone is on the same page. For now I just set it to nolint

Reply:
actually these are used, not sure why the linter is picking them up as unused

Reply:
Yes disabling linter for test packages or even test files (UT) makes sense.

Reply:
Added an ignore to all the files in e2etest and cleaned up files in e2etest that had manual nolint lines